docs/api.rst

```diff
@@ -6036,6 +6059,22 @@ ApplicationRoleConnectionMetadata
 .. autoclass:: ApplicationRoleConnectionMetadata
     :members:
 
+OnboardingPrompt
+~~~~~~~~~~~~~~~~
+
+.. attributetable:: OnboardingPrompt
+
+.. autoclass:: OnboardingPrompt
+    :members:
+
+OnboardingPromptOption
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. attributetable:: OnboardingPromptOption
+
+.. autoclass:: OnboardingPromptOption
+    :members:
+
```

Comment:
these two should be put under Discord Models, I think

Reply:
c2ce2cc76acec06a265cfdc38e811315aeceb856